speed-dreams/source-2.2.3/src/libs/tgfclient/tgfclient.h

```diff
@@ -268,12 +271,35 @@ typedef struct CheckBoxInfo
     void        *userData;      /**< Associated user data */
 } tCheckBoxInfo;
 
+// SIMULATED DRIVING ASSISTANCE CHANGE: Added radiobutton information
+/// @brief Radio-button call-back information
+typedef struct RadioButtonInfo
+{
+    bool  checked;  /**< Radio-button state */
+    int   selected; /**< Radio-button-id selected from the parent radio-button-list */
+    void* userData; /**< Associated user data */
```

Comment:
These are public members and should follow the naming standards
```suggestion
    bool  Checked;  /**< Radio-button state */
    int   Selected; /**< Radio-button-id selected from the parent radio-button-list */
    void* UserData; /**< Associated user data */
```